packages/neutrino-lint-base/src/index.js

```diff
@@ -22,7 +22,7 @@ module.exports = neutrino => {
           root: true,
           plugins: ['babel'],
           baseConfig: {},
-          envs: ['es6'],
+          envs: [{'es6': true}],
```

Comment:
This property is still used by ESLint's CLIEngine, so this line needs to stay `envs: ['es6'],`.